### PR TITLE
add subscription services to templates

### PR DESCRIPTION
**Pull request by grs**

### CODE REVIEW

File: templates/include/subserv.jsonnet [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "metadata": {
+          "labels": {
+            "name": "subserv"
+          }
+        },
+        "spec": {
+          "containers": [
+            {
+              "image": image_name,
+              "name": "master",
```
Comment (lulf) [inline templates/include/subserv.jsonnet]: Should we call this subserv as well?

File: templates/include/subserv.jsonnet [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              "ports": [
+                {
+                  "containerPort": 5672,
```
Comment (lulf) [inline templates/include/subserv.jsonnet]: One thing I started doing was to add a 'name' attribute to each port, like 'name': 'amqp'

File: templates/include/subserv.jsonnet (hunk 1 above)
Comment (grs) [inline templates/include/subserv.jsonnet]: done

File: templates/include/subserv.jsonnet (hunk 2 above)
Comment (grs) [inline templates/include/subserv.jsonnet]: done

**Pull request merged**